doc/cookbook/splitting.md

```diff
@@ -66,3 +66,146 @@ $ pachctl put-file users master -c -f user_data.json --split json --target-file-
 # bytes chunk into the split files.
 $ pachctl put-file users master -c -f user_data.txt --split line --target-file-bytes 100
 ```  
+
+## Specifying Header/Footer
+
+Additionally, if your data has a common header or footer, you can specify these
+manually via `pachctl put-header` or `pachctl put-footer`. This is helpful for CSV data.
+
+To do this, you'll need to specify the header/footer on the parent directory of your data. Below we have an example of splitting a CSV with a header, then setting the header explicitly. Notice that once we've set the header, whenever we get a file under that directory, the header is applied. You can still use glob patterns to get all the data under the directory, and in that case the header is still applied.
```

Comment:
Out of curiosity, how does header/footers work if you have nested dirs? Does it apply to files recursively or break if you try to get-file on a dir with or without a header embedded?